SharpPcap/LibPcap/LibPcapLiveDevice.cs

```diff
@@ -131,10 +131,7 @@ public virtual uint KernelBufferSize
         /// </param>
         public override void Open(DeviceModes mode = DeviceModes.None, int read_timeout = 1000, MonitorMode monitor_mode = MonitorMode.Inactive, uint kernel_buffer_size = 0, RemoteAuthentication credentials = null)
         {
-            if (credentials == null)
-            {
-                credentials = Interface.Credentials;
-            }
+            var cred = credentials ?? Interface.Credentials;
```

Comment:
can we just override the credentials input argument? I like the new null syntax though.  I've always been a big fan of using full names like 'credentials' vs. 'cred' to help with search and readability.